setup.py

```diff
@@ -25,9 +25,13 @@ def readme():
           'pyhocon',
           'mock',
           'arrow',
-          'nose'
+          'nose',
+          'requests',
+          'thrift==1.0.0-py3',
+          'munch'
       ],
       entry_points={'console_scripts': ['herc = herc.webservice:main']},
+      dependency_links = [urllib.parse.urljoin('file:', os.path.join(os.getcwd(), 'thrift-1.0.0-py3.tar.gz'))],
```

Comment:
That's my terrible doing. I added a tarball of my Python3 build of the Thrift library to the repo; this is some magic to reference that when attempting to satisfy the Thrift dependency.

It should be added to the package files if it isn't already.